backend/db.py

```diff
@@ -95,19 +95,24 @@ def get_all_subscribers(self):
         collection = self.db["subscribers"]
         return list(collection.find())
 
-    def add_subscriber(self, phone_number):
+    def add_subscriber(self, phone_number) -> bool:
         if not self.is_connected:
             logging.warning("not connected to database")
-            return
+            return False
         collection = self.db["subscribers"]
-        collection.insert_one({"phone_number": phone_number})
+        response = collection.update_one({"phone_number": phone_number}, {"upsert": True})
```

Comment:
Getting error: `ValueError: update only works with $ operators`. 

This seems to work: `response = collection.update_one({"phone_number": phone_number}, { "$set": {"phone_number": phone_number}  }, upsert=True)`